project1/application.py

```diff
@@ -21,9 +22,17 @@
 
 # Set up database
 engine = create_engine(os.getenv("DATABASE_URL"))
-db = scoped_session(sessionmaker(bind=engine))
+db = SQLAlchemy(app)
+
+
+class user(db.Model):
+    id = db.Column(db.Integer, primary_key=True)
+    username = db.Column(db.String(80))
+    email = db.Column(db.String(120), unique=True)
+    password = db.Column(db.String(80))
+
 
-names = {}
+db.create_all()
```

Comment:
A good practice is to create the db in am main function & running application separately on the console. Just a reference for future.